compiler/cgen.nim

```diff
@@ -33,6 +33,12 @@ import std/strutils except `%`, addf # collides with ropes.`%`
 from ic / ic import ModuleBackendFlag
 import std/[dynlib, math, tables, sets, os, intsets, hashes]
 
+const
+  # we use some ASCII control characters to insert directives that will be converted to real code in a postprocessing pass
+  postprocessDirStart = '\1'
```

Comment:
Seems to be never generated?

Reply:
It is generated in the first line of genPostprocessDir.